### PR TITLE
alure2: init at unstable-2019-09-21

### PR DESCRIPTION
**Pull request by McSinyx**
###### Motivation for this change
I am trying to add the new major release of alure, which has C++11 API instead of the previous C API.  This should not replace alure 1 since the two are incompatible.

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Tested compilation and usage of examples
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/alure2/default.nix
Comment (aanderse) [inline pkgs/development/libraries/alure2/default.nix]: A new commit has been pushed. If you would like to update to that feel free.

**Review by aanderse — CHANGES_REQUESTED** [review]
Hi. Thanks for contributing a new package to nixpkgs! We're especially excited you want to join the team and maintain the package :tada:

The package looks good. Just a minor nitpick or two. I'll build this once I get a chance and we can get this merged.

**Comment by McSinyx** [comment]
Thank you for the review, I've updated the nix file accordingly.

**Review by aanderse — APPROVED** [review]
LGTM :+1: Thanks again :tada:

**Comment by aanderse** [comment]
Ah sorry... @McSinyx I forgot, you're supposed to add yourself to the list of maintainers in a separate commit! Please do that and then we're ready to merge. Thanks!

**Comment by McSinyx** [comment]
Thanks, I've separated the change into 2 commits.

**Pull request merged**
